assets/translations/main.json

```diff
@@ -408,5 +415,29 @@
 	"vscode.file-open.backup-failed": "Backup failed: this is not a .tldr file.",
 	"vscode.file-open.dont-show-again": "Don’t ask again",
 	"cursor-chat.type-to-chat": "Type to chat…",
-	"app.loading": "Loading tldraw…"
+	"app.loading": "Loading tldraw…",
+	"handle.resize-top": "Resize top",
+	"handle.resize-bottom": "Resize bottom",
+	"handle.resize-left": "Resize left",
+	"handle.resize-right": "Resize right",
+	"handle.resize-top-left": "Resize top left",
+	"handle.resize-top-right": "Resize top right",
+	"handle.resize-bottom-left": "Resize bottom left",
+	"handle.resize-bottom-right": "Resize bottom right",
+	"handle.rotate.top_left_rotate": "Rotate top left",
+	"handle.rotate.top_right_rotate": "Rotate top right",
+	"handle.rotate.bottom_left_rotate": "Rotate bottom left",
+	"handle.rotate.bottom_right_rotate": "Rotate bottom right",
+	"handle.rotate.mobile_rotate": "Rotate",
+	"handle.crop.top": "Crop top",
+	"handle.crop.bottom": "Crop bottom",
+	"handle.crop.left": "Crop left",
+	"handle.crop.right": "Crop right",
+	"handle.crop.top-left": "Crop top left",
+	"handle.crop.top-right": "Crop top right",
+	"handle.crop.bottom-left": "Crop bottom left",
+	"handle.crop.bottom-right": "Crop bottom right",
```

Comment:
truly hope that screen readers aren't reading the overlay UI

Reply:
The thing is that it's part of the a11y tree by definition (even if tabbing over things skips over things at the moment):

![Screenshot 2025-05-06 at 15 04 10](https://github.com/user-attachments/assets/2e7cb9ef-233f-4c82-b453-d3e6ed68e323)

Reply:
example here is that the "Clone note" buttons are for the nibs